src/components/Footer/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import SingFooter from '../SingFooter/index'
+import { AnchorLink, Parent, List, FooterStyle } from './style';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import { Container } from '@mui/system';
+
+
+
+ const Footer: React.FC= () => {
+  return (
+    <><Container>
+      <SingFooter/>
+    <FooterStyle>
```

Comment:
please use prettier for code formatting
https://www.youtube.com/watch?v=h3PJjP0nE98